app/src/ingestion/scrape_edd_web.py

```diff
@@ -0,0 +1,67 @@
+import json
+import logging
+import os
+from pprint import pprint
+
+from scrapy.crawler import CrawlerProcess
+from scrapy.utils.project import get_project_settings
+
+logger = logging.getLogger(__name__)
+
+
+# Typically, env variable SCRAPY_PROJECT should be set when calling this script.
+# The SCRAPY_PROJECT env variable refers to the projects defined in scrapy.cfg
+if "SCRAPY_PROJECT" not in os.environ:
+    # The alternative to this is to set the `default` project in scrapy.cfg and run:
+    # `scrapy crawl edd_spider -o edd_scrapings.json` instead of this script.
+    # This script is useful for postprocessing the json output.
+    os.environ["SCRAPY_PROJECT"] = "edd"
+
+OUTPUT_JSON = "edd_scrapings.json"
+settings = get_project_settings()
+settings["FEEDS"] = {
+    OUTPUT_JSON: {"format": "json"},
+}
+process = CrawlerProcess(settings)
+pprint(process.settings.copy_to_dict())
+
+# Remove the output file if it already exists so that Scrapy doesn't append to it
+if OUTPUT_JSON in os.listdir():
+    os.remove(OUTPUT_JSON)
+
+# 'edd_spider' is the name of one of the spiders (see EddSpider.name)
+process.crawl("edd_spider")
+process.start()  # the script will block here until the crawling is finished
+
+logger.info("Scraping results saved to %s", OUTPUT_JSON)
+# Postprocess the JSON output for readability
+with open(OUTPUT_JSON, "r", encoding="utf-8") as raw_json:
+    data = json.load(raw_json)
+
+    if "DEBUG_SCRAPINGS" in os.environ:
```

Comment:
Should this `if` statement be lifted up?

Reply:
Yes!